fetch-polyfill.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
+// fetch-polyfill.js
```

Comment:
required by viem: https://viem.sh/docs/compatibility
will be removed after upgrading node to v18